### ISSUE TITLE
Wrong default font when show CJK characters

### ISSUE THREAD

**Issue by chenxuuu** [AvaloniaUI/Avalonia#12349]
**Describe the bug**

Wrong default font when show CJK characters

for example my computer use Chinese but it show me Japanese font `Yu Gothic UI`

**To Reproduce**

`Microsoft YaHei` is a Chinese font in Windows

```xml
<!--wrong--><TextBlock Text="包直入今骨角房" FontSize="50" />
<!--right--><TextBlock Text="包直入今骨角房" FontSize="50" FontFamily="Microsoft YaHei"/>
```

![图片](https://github.com/AvaloniaUI/Avalonia/assets/10357394/8084620b-4fe1-4087-ab26-1681eb54c07c)

( Project: [AvaloniaApplication3.zip](https://github.com/AvaloniaUI/Avalonia/files/12180632/AvaloniaApplication3.zip) )

**Expected behavior**

It shoud use Chinese font

**Screenshots**

see above

**Desktop (please complete the following information):**

- OS: Windows 10
- Version 22H2 19045.3208
- Language `中文(简体，中国)` Chinese (Simplified, China)

and same on my friend's computer

- OS: Windows 11
- Version 22H2

**Additional context**

**Comment by hez2010** [comment]
Can you check if your default administrative locale codepage is set to Simplified Chinese (936)?

**Comment by chenxuuu** [comment]
@hez2010 I think it is... all default value, never changed

![图片](https://github.com/AvaloniaUI/Avalonia/assets/10357394/1a771e8a-7a15-4938-8ed4-012f2ccdd5b7)
![图片](https://github.com/AvaloniaUI/Avalonia/assets/10357394/e2ad9f15-50ea-4b4d-b58a-06f65c81f252)
![图片](https://github.com/AvaloniaUI/Avalonia/assets/10357394/064a9deb-52aa-4c14-83b6-3f8bb48e51c7)

**Comment by chenxuuu** [comment]
It seems fallback to `Yu Gothic UI` font, strange

**Comment by chenxuuu** [comment]
still wrong in 11.0.6

**Comment by Gillibald** [comment]
Font fallbacks are implemented via DirectWrite on Windows. So the result is defined by the OS. You can define a custom fallback so the FontManager selects a different font

**Comment by YexuanXiao** [comment]
> Font fallbacks are implemented via DirectWrite on Windows. So the result is defined by the OS. You can define a custom fallback so the FontManager selects a different font

The problem seems to have existed for over 4 years. Whether or not DirectWrite returns the correct result, I think Avalonia should fix it. I would like to offer some help in any way I can to resolve the issue. Can you give me some guidance (e.g. how Avalonia gets the default font from DirectWrite)?

**Comment by Gillibald** [comment]
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Skia/Avalonia.Skia/FontManagerImpl.cs#L62

We rely on Skia here which uses DirectWrite to match characters. You can always define FontFallbacks manually in your AppBuilder

**Comment by YexuanXiao** [comment]
> > https://github.com/AvaloniaUI/Avalonia/blob/master/src/Skia/Avalonia.Skia/FontManagerImpl.cs#L62
> > We rely on Skia here which uses DirectWrite to match characters. You can always define FontFallbacks manually in your AppBuilder
> > #7089
> 
> I checked the source code of SkiaSharp and Skia, and it turns out that using the ietfLanguageTag is sufficient to solve the issue, rather than *LetterISOLanguageName. SkiaSharp and Skia mark the parameter as using bcp47, which is generally equivalent to ietf. Therefore, there’s no need to use *LetterISOLanguageName.

BCP47 spec: https://www.[rfc-editor.org/rfc/bcp/bcp47.txt](https://www.rfc-editor.org/rfc/bcp/bcp47.txt)

**Comment by Gillibald** [comment]
Feel free to adjust the language tag being used

**Comment by chenxuuu** [comment]
fixed after in Avalonia 11.2.x beta, I checked it works perfect

**Comment by YexuanXiao** [comment]
> fixed after in Avalonia 11.2.x beta, I checked it works perfect

Good news.
